### PR TITLE
Add exporter dbzm plugin to voyager repo

### PR DESCRIPTION
**Pull request by makalaaneesh**
- Moved debezium-server voyager plugin from https://github.com/yugabyte/debezium/tree/voyager-main to this repo
- Modified installer script to build and package debezium-server when --install-from-local-source
    - Added a flag `rebuild-voyager-local` that will only rebuild voyager(go install) and debezium-server plugin (maven install + copy jars to already installed debezium-server. This will make it easy to test incremental changes to voyager.

### CODE REVIEW

File: installer_scripts/install-yb-voyager [hunk 1]
```diff
@@ -140,6 +161,15 @@ centos_main() {
 ubuntu_main() {
 	check_binutils_version
 	check_java
+	if [ "${REBUILD_VOYAGER_LOCAL}" == "true" ]
+	then
+		unset GOROOT
```
Comment (amit-yb) [inline installer_scripts/install-yb-voyager]: Try without unsetting the GOROOT.

File: installer_scripts/install-yb-voyager [hunk 2]
```diff
@@ -228,10 +289,166 @@ install_debezium_server() {
 	rm ${debezium_server_filename}
 }
 
+check_install_maven() {
+	MIN_REQUIRED_MAVEN_VERSION='3.8.4'
+	MAVEN_VER=$(mvn -version 2>&1 | grep "Apache Maven" | awk '{print $3}')
+    version_ok=$(version_satisfied "$MIN_REQUIRED_MAVEN_VERSION" "$MAVEN_VER")
+	if [[ $version_ok -eq 1 ]]
+	# if ([ -n "$MAVEN_VER" ] && (( 10#${MAVEN_VER} >= 10#${MIN_REQUIRED_MAVEN_VERSION} )) )
```
Comment (amit-yb) [inline installer_scripts/install-yb-voyager:288]: Remove commented code.

File: installer_scripts/install-yb-voyager [hunk 3]
```diff
@@ -228,10 +289,166 @@ install_debezium_server() {
...
+        then
```
Comment (amit-yb) [inline installer_scripts/install-yb-voyager:289]: Fix indentation.

File: installer_scripts/install-yb-voyager [hunk 4]
```diff
@@ -228,10 +289,166 @@ install_debezium_server() {
 	rm ${debezium_server_filename}
 }
 
+check_install_maven() {
+	MIN_REQUIRED_MAVEN_VERSION='3.8.4'
+	MAVEN_VER=$(mvn -version 2>&1 | grep "Apache Maven" | awk '{print $3}')
+    version_ok=$(version_satisfied "$MIN_REQUIRED_MAVEN_VERSION" "$MAVEN_VER")
```
Comment (amit-yb) [inline installer_scripts/install-yb-voyager:286]: Fix indentation. There are mix of tabs and spaces at the beginning.

File: installer_scripts/install-yb-voyager [hunk 5]
```diff
@@ -228,10 +289,166 @@ install_debezium_server() {
...
+	mvn -ntp clean install -Dquick -f ${pom} 1>&2
+
+	popd > /dev/null
+}
+
+assemble_debezium_server_voyager_local() {
+	output "Assembling debezium-server voyager..." "$1"
+	if [ "$1" = "2x" ] 
+	then
+		pom="pom.xml"
```
Comment (amit-yb) [inline installer_scripts/install-yb-voyager:370]: Consider having named variables for `pom.xml` and `195pom.xml`

File: installer_scripts/install-yb-voyager [hunk 6]
```diff
@@ -228,10 +289,166 @@ install_debezium_server() {
...
+	popd > /dev/null
+}
+
+package_debezium_server_local(){
+	output "Packaging debezium server..."
+	pushd `git rev-parse --show-toplevel` > /dev/null
+
+	cd debezium-server-voyager/target
+	tar -xvzf "debezium-server-voyager-dist-${DEBEZIUM_LOCAL_VERSION}.Final.tar.gz" 1>&2
+	rm "debezium-server-voyager-dist-${DEBEZIUM_LOCAL_VERSION}.Final.tar.gz"
```
Comment (amit-yb) [inline installer_scripts/install-yb-voyager:401]: Use named identifiers to refer to the paths.

File: installer_scripts/install-yb-voyager [hunk 7]
```diff
@@ -228,10 +280,166 @@ install_debezium_server() {
...
+		sudo mv yb-debezium-maven-3.8.4 /opt/yb-voyager/
+		rm apache-maven-3.8.4-bin.tar.gz
+	fi
+	export PATH=/opt/yb-voyager/yb-debezium-maven-3.8.4/bin:$PATH
+}
+
+clean_debezium(){
+	pushd `git rev-parse --show-toplevel` > /dev/null
+	rm -rf debezium-server-voyager/target
+	popd > /dev/null
```
Comment (amit-yb) [inline installer_scripts/install-yb-voyager:315]: How about saving the REPO_ROOT at the beginning of the script and then directly constructing abs paths in the commands:
`rm -rf $REPO_ROOT/debezium-server-voyager/target`

**Review by amit-yb — APPROVED** [review]
LGTM with minor comments.

File: installer_scripts/install-yb-voyager (hunk 7 above)
Comment (makalaaneesh) [reply installer_scripts/install-yb-voyager]: Set REPO_ROOT=git rev-parse --show-toplevel
In cases where we just have to run some rm/mv commands, I've directly used REPO_ROOT. 
In cases where we need to be in the directory (to run mvn for example), I've used pushd `$REPO_ROOT/x/y/z.`

**Pull request merged**